client/my-sites/stats/stats-module/connected-list.jsx

```diff
@@ -168,11 +230,25 @@ export default connect( ( state, ownProps ) => {
 	const siteId = getSelectedSiteId( state );
 	const siteSlug = getSiteSlug( state, siteId );
 	const { statType, query } = ownProps;
+	let priorPeriodData;
+	let priorPeriodQuery;
+	let requestingPriorPeriod;
+
+	// Fetch Prior Period
+	if ( ownProps.showPriorPeriodToggle && ! ownProps.summary ) {
+		priorPeriodQuery = merge( {}, query );
+		priorPeriodQuery.date = moment( query.date ).locale( 'en' ).subtract( 1, 'd' ).format( 'YYYY-MM-DD' );
```

Comment:
We could write this something like:

```js
priorPeriodQuery = {
  ...query,
  date: moment( query.date ).locale( 'en' ).subtract( 1, 'd' ).format( 'YYYY-MM-DD' )
}
```

I know, I've read too much of Dan Abramov's code :P